youtube_dl/extractor/rai.py

```diff
@@ -240,20 +222,32 @@ class RaiPlayLiveIE(RaiBaseIE):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, display_id)
+        media = self._download_json(
+            '%s.json' % urljoin(RaiBaseIE._BASE_URL, 'dirette/' + display_id),
+            display_id, 'Downloading channel JSON')
+
+        title = media['name']
+        video = media['video']
+        video_id = media['id'].replace('ContentItem-', '')
 
-        video_id = self._search_regex(
-            r'data-uniquename=["\']ContentItem-(%s)' % RaiBaseIE._UUID_RE,
-            webpage, 'content id')
+        relinker_info = self._extract_relinker_info(video['content_url'], video_id)
+        self._sort_formats(relinker_info['formats'])
 
-        return {
-            '_type': 'url_transparent',
-            'ie_key': RaiPlayIE.ie_key(),
-            'url': 'http://www.raiplay.it/dirette/ContentItem-%s.html' % video_id,
+        info = {
             'id': video_id,
             'display_id': display_id,
+            'title': self._live_title(title) if relinker_info.get(
+                'is_live') else title,
+            'alt_title': media.get('subtitle'),
+            'description': media.get('description'),
+            'uploader': strip_or_none(media.get('channel')),
+            'creator': strip_or_none(media.get('editor')),
+            'duration': parse_duration(video.get('duration')),
```

Comment:
is there a URL where the `duration` is available.

Reply:
There should be... I will try to find one and add it as a test, thanks!

Reply:
Yes, the one in the test (<http://www.raiplay.it/video/2014/04/Report-del-07042014-cb27157f-9dd0-4aee-b788-b1f67643a391.html>) has it.

Reply:
this URL you have provided is not related to this part of the code(`RaiPlayLiveIE`).

Reply:
Whoops, I accidentally confused them with RaiPlayIE, thanks!  I have just double-checked all such URL via <https://www.raiplay.it/dirette/> and none of them have an `alt_title` or `duration` so I removed both of them. This should be addressed by https://github.com/ytdl-org/youtube-dl/pull/27077/commits/2ea4cfb5f8d7a882c2e5c7aedd60af18a36881e5